Fix: error is raised on lambda

```bash
[ERROR] KeyError: 'prices'
Traceback (most recent call last):
  File "/var/task/main.py", line 14, in lambda_handler
    trader = RealTrader(operation='live')
  File "/var/task/src/real_trader.py", line 30, in __init__
    super(RealTrader, self).__init__(operation=operation, days=60)
  File "/var/task/src/trader.py", line 48, in __init__
    result: Dict[str, str] = self._candle_loader.run()
  File "/var/task/src/candle_loader.py", line 32, in run
    tradeable = self.client_manager.call_oanda('is_tradeable')['tradeable']
  File "/var/task/src/client_manager.py", line 176, in call_oanda
    return method_dict.get(method)(**kwargs)
  File "/var/task/src/clients/oanda_client.py", line 58, in request_is_tradeable
    tradeable = response['prices'][0]['tradeable']
```

--- a/src/clients/oanda_client.py
+++ b/src/clients/oanda_client.py
@@ -205,6 +205,7 @@ class OandaClient():
             response = self.__api_client.request(obj)
         except V20Error as error:
             LOGGER.error({f'[{sys._getframe().f_back.f_code.co_name}] V20Error': error})
+            LOGGER.info({f'[{sys._getframe().f_back.f_code.co_name}] dir(error)': dir(error)})
             # error.msg
             return {'error': error.code}
         except requests.exceptions.ConnectionError as error:

